tests/test_schema.yaml

```diff
@@ -634,6 +634,8 @@ schemas:
   on_parent_delete_cascade: false
   extends:
   - base
+  metadata:
+    disable_substr_search: true
```

Comment:
please add [documentation](https://github.com/cloudwan/gohan/blob/master/docs/schema.md#Metadata) for this feature